### PR TITLE
Type definitions for Victor.js 0.8.0 and Matter-js.js 0.2.0

### PR DESCRIPTION
**Pull request by ghost**
Those are type definitions for vector math library Vector.js 0.8.0

### CODE REVIEW

**Comment by vvakame** [comment]
*matter/matter.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/matter
  * http://bower.io/search/?q=matter
  * others?
* [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (matter/matter-tests.ts or others)
* [ ] pass the Travic-CI test?

---

*victor/victor.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/victor
  * http://bower.io/search/?q=victor
  * others?
* [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (victor/victor-tests.ts or others)
* [ ] pass the Travic-CI test?

File: matter/matter.d.ts [hunk 1]
```diff
@@ -0,0 +1,1509 @@
+// Type definitions for Matter.js 0.8.0
+// Project: http://brm.io/matter-src/
```
Comment (vvakame) [inline matter/matter.d.ts]: this url returns 404.

**Comment by ivanegegia** [comment]
I've corrected naming;
Added test files;
Made some modifications;
Travis CI test are passing, as far as I know correctly where to look :) .

**Comment by vvakame** [comment]
*matter-js/matter-js.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/matter-js
  * http://bower.io/search/?q=matter-js
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (matter-js/matter-js-tests.ts or others)
* [ ] pass the Travic-CI test?

---

*victor/victor.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/victor
  * http://bower.io/search/?q=victor
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (victor/victor-tests.ts or others)
* [ ] pass the Travic-CI test?

File: .gitignore [hunk 2]
```diff
@@ -41,3 +41,4 @@ _infrastructure/tests/build
 node_modules
 
 .sublimets
+.settings/launch.json
```
Comment (vvakame) [inline .gitignore]: what's this?

Comment (ivanegegia) [inline .gitignore]: This if file created by Visual Studio Code(the new editor), I've opened DefinitelyTyped folder with it, so it created one(.settings/launch.json). 
Shell I remove it or something?

Comment (vvakame) [inline .gitignore]: i see. np!

**Comment by vvakame** [comment]
@ivanegegia thanks mate.

**Pull request merged**